azure-pipelines.yml

```diff
@@ -23,13 +26,45 @@ jobs:
       ./docker_push.sh
     workingDirectory: build_container
     env:
+      OS_FAMILY: linux
+      IMAGE_ARCH: "amd64 arm64"
       SOURCE_BRANCH: $(Build.SourceBranch)
       DOCKERHUB_USERNAME: $(DockerUsername)
       DOCKERHUB_PASSWORD: $(DockerPassword)
       GCP_SERVICE_ACCOUNT_KEY: $(GcpServiceAccountKey)
 
-- job: generate_toolchains
-  dependsOn: "build_container"
+- job: build_container_windows
+  strategy:
+    matrix:
+      windows2019:
```

Comment:
added this structure as we might get more flavors of Windows toolchains in the future to enable development on different versions of windows, not sure, can revert this if we think it's not going to be necessary

Reply:
we might need different `vmImage`s for each of the different windows flavors (b/c of the unfortunate realities of Windows containers), so this might not be possible to do in a matrix like this unless AZP lets you use a template that specifies a different `vmImage` as well as build steps

Reply:
This is fine, I don't think merging this two job makes a lot of sense while AZP have yaml templates.